From 32ac5f59d98764af99bb1bdf4f6fa2bfc8610327 Mon Sep 17 00:00:00 2001
From: Huang Shijie <b32955@freescale.com>
Date: Wed, 8 Feb 2012 11:19:07 +0800
Subject: [PATCH 0974/2463] ENGR00174021 MX6Q/ARCH : fix the section mismatch

add __init to the gpmi_nand_platform_init() to make this
function store in the init.text section.

Signed-off-by: Huang Shijie <b32955@freescale.com>
---
 arch/arm/mach-mx6/board-mx6q_arm2.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arch/arm/mach-mx6/board-mx6q_arm2.c b/arch/arm/mach-mx6/board-mx6q_arm2.c
index 04025bc..dc7698b 100644
--- a/arch/arm/mach-mx6/board-mx6q_arm2.c
+++ b/arch/arm/mach-mx6/board-mx6q_arm2.c
@@ -250,7 +250,7 @@ static const struct esdhc_platform_data mx6_arm2_sd4_data __initconst = {
 	.platform_pad_change	= plt_sd4_pad_change,
 };
 
-static int gpmi_nand_platform_init(void)
+static int __init gpmi_nand_platform_init(void)
 {
 	iomux_v3_cfg_t *nand_pads = NULL;
 	u32 nand_pads_cnt;
-- 
1.7.7.4

